src/components/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export function Sum() {}
+import React from 'react';
+
+export const Sum = ({ a, b }) => (
+  <p>{`Sum of ${a} and ${b} is ${a + b}`}</p>
+);
```

Comment:
```suggestion
export const Sum = ({ a, b }) => (
  <p>
    {`Sum of ${a} and ${b} is ${a + b}`}
   </p>
);
```
I think this sign would be more readable

Reply:
agree, thanks! fixed